flashinfer_bench/compile/runnable.py

```
@@ -65,30 +74,160 @@ def __init__(
-        return ret
+        else:
+            # len(ret) == 0
+            return None
+
+    @cached_property
+    def _arg_names(self) -> List[str]:
+        """Get the order of arguments for the underlying function."""
+        if self.metadata.definition is None:
+            raise ValueError(
+                "When calling in keyword passing style, metadata.full_definition must " "be set."
+            )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)